### PR TITLE
Wrote on Product' Views, Model, Service and Debugged Business Model's commits' bugs.

### PR DESCRIPTION
**Pull request by jacques-twizeyimana**

### CODE REVIEW

**Review by veritem — APPROVED** [review]

**Review by thepatrickniyo — APPROVED** [review]

File: src/com/customify/client/services/ProductService.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.customify.client.services;
+
+import com.customify.client.Common;
+import com.customify.server.models.ProductModel;
```
Comment (thepatrickniyo) [inline src/com/customify/client/services/ProductService.java]: It is not a good practice for a client to directly access backend. Note that the communication should be a `tcp`

File: src/com/customify/client/views/ProductView.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<<<<<<< HEAD
```
Comment (thepatrickniyo) [inline src/com/customify/client/views/ProductView.java]: Remove that also

File: src/com/customify/client/views/ProductView.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        this.socket = socket;
+    }
+
+    public void getAll() {
+        System.out.println("++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++");
+        System.out.println("\t\tHere is a list of products registered so far");
+        System.out.println("++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++");
+
+        }
+=======
```
Comment (thepatrickniyo) [inline src/com/customify/client/views/ProductView.java]: You pushed with a conflict there

File: src/com/customify/client/views/ProductView.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+package com.customify.client.views;public class ProductView {
+>>>>>>> 112ef6f5f3c53be2677a51a1925833905190461f
```
Comment (thepatrickniyo) [inline src/com/customify/client/views/ProductView.java]: The same case here

**Review by thepatrickniyo — APPROVED** [review]

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Good work @jtwizeyimana! work on those feedback ASAP, make sure every thing runs well and we are ready to go

File: src/com/customify/client/services/ProductService.java (hunk 1 above)
Comment (jacques-twizeyimana) [reply src/com/customify/client/services/ProductService.java]: Ok, I will create a product format

**Review by veritem — CHANGES_REQUESTED** [review]
Please remove all conflicts

**Comment by SauveJeanLuc** [comment]
Conflicts Resolved, Pull Request can now be merged @makuzaverite @PatrickNiyogitare28

**Pull request merged**